.github/workflows/deploy-to-control-plane-staging.yml

```
@@ -1,32 +1,94 @@
-          ref: master    # Explicitly checkout master branch
+          fetch-depth: 0
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup-environment
+        with:
+          token: ${{ secrets.CPLN_TOKEN_STAGING }}
+          org: ${{ vars.CPLN_ORG_STAGING }}
+
+      - name: Build Docker Image
+        id: build
+        uses: ./.github/actions/build-docker-image
```

actionlint (1.7.4)

missing input "PR_NUMBER" which is required by action "Build Docker Image" defined at "./.github/actions/build-docker-image". all required inputs are "PR_NUMBER", "app_name", "commit", "org"

(action)